back/config/environments/test.rb

```diff
@@ -65,11 +64,14 @@
   # Tell Active Support which deprecation messages to disallow.
   config.active_support.disallowed_deprecation_warnings = []
 
+  # Raises error for missing translations.
+  # config.i18n.raise_on_missing_translations = true
+
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 
-  # Raise error when a before_action's only/except options reference missing actions
-  config.action_controller.raise_on_missing_callback_actions = false
+  # Raise error when a before_action's only/except options reference missing actions.
+  config.action_controller.raise_on_missing_callback_actions = true
```

Comment:
I'd suggest setting it back to false and maybe bringing it up with the CDM. Otherwise, configuring a `verify_policy_scoped` callback in the `ApplicationController` doesn't make much sense.